docs/nx-cloud/features/split-e2e-tasks.md

```diff
@@ -34,6 +34,13 @@ nx add @nx/playwright
 nx add @nx/jest
 ```
 
+{% /tab %}
+{% tab label="Gradle" %}
+
+```shell {% skipRescope=true %}
+nx add @nx/gradle
+```
+
```

Comment:
Can we leave out the gradle changes until we've tested it on Ocean and ensure that it's production ready?